CHANGELOG.md

```diff
@@ -2,6 +2,95 @@
 
 ## 28
 
+### Call Caching
```

Comment:
ToL This is pretty beefy for the Changelog.  Are there more detailed one-off docs that could describe individual topics?